templates/home.html

```diff
@@ -29,11 +29,15 @@
 
         <div class="col center">
             <div class="in">
-                <h2>Shogun talks</h2>
+                <h2>Google Summer of Code 2015</h2>
                 <p align="justify">
-                    SHOGUN was presented in July at the <a href="https://ep2014.europython.eu/en/schedule/sessions/103/">Europython conference</a> in Berlin and in August at the <a href="http://hunch.net/~nyoml/">Open Machine Learning Workshop</a> in New York, both times by Heiko Strathmann. These talks are of interest both to people that do not know about SHOGUN and are interested in using it or just getting an introduction, as well as for people with prior experience using SHOGUN.
+                    <strong>We have applied as a mentoring org.<br></strong>
+                    Our list of project ideas this year is full of exciting and fun
+                    projects. There is also a new guide for prospective students. Let's now keep
+                    our fingers crossed until accepted mentoring orgs are announced!
                 </p>
             </div>
+            <p class="nom"><a href="/page/Events/gsoc2014/"><img src="/static/design/btn-more.gif" alt="More" /></a></p>

```

Comment:
Sorry! Did not see that in the flood.

It would be good to have a new page, just like this one:
http://www.shogun-toolbox.org/page/Events/gsoc2014

It would be good to have the "we are applying" part on there for now, almost the same as last year

And then links to 
https://github.com/shogun-toolbox/shogun/wiki/Google-Summer-of-Code-2015-projects
https://github.com/shogun-toolbox/shogun/wiki/Getting-involved
https://github.com/shogun-toolbox/shogun/wiki/GSoC-follow-up-blog-posts